config_create.go

```diff
@@ -39,6 +39,9 @@ func init() {
 	privileged := &defaultConfig.Machines[0].Spec.Privileged
 	configCreateCmd.PersistentFlags().BoolVar(privileged, "privileged", *privileged, "Create privileged containers")
 
+	persistent := &defaultConfig.Machines[0].Spec.Persistent
```

Comment:
I think we should make `persistent = true` the default and not even allow the creation non "persistent" VMs.

Currently we do:

```
footloose create
...
footloose delete
```

Instead of using `--rm` in `create`, let's move the container deletion to `delete`, so `delete` does `kill` + `rm`. This way we can keep the current behaviour and allow `start`/`stop` to work.

Reply:
Removed the flag to make containers persistent by default on last commit.